apps/backend/src/utils/nginx_parser.py

```
@@ -0,0 +1,553 @@
+                    
+            except Exception as e:
+                results['summary']['parse_errors'] += 1
+                results['errors'].append(f"Error parsing {conf_file.name}: {str(e)}")
+        
+        # Deduplicate services
+        results['summary']['services'] = sorted(set(results['summary']['services']))
+        
+    except Exception as e:
+        results['errors'].append(f"Error accessing directory: {str(e)}")
+    
+    return results
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)